src/styles/star.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background: url(../images/star.svg) no-repeat center;
+    display: flex;
+
+    &:nth-child(-n + 4) {
```

Comment:
The use of `:nth-child(-n + 4)` is correct for selecting the first four children, but ensure that this logic aligns with your design requirements. If you intend to highlight a different number of stars based on a condition, make sure this selector is dynamic or correctly reflects the intended behavior.